resources/event-sources/templates/uninstall.yaml

```diff
@@ -72,14 +72,14 @@ spec:
     metadata:
       annotations:
         sidecar.istio.io/inject: "false"
-      name: {{ .Release.Name }}-rc-delete
+      name: {{ .Release.Name }}-cr-delete
       labels:
-        app: "event-sources-delete"
+        kyma-project.io/app: "eventing-uninstall"
```

Comment:
how about: kyma-project.io/uninstall: eventing

one could then get all uninstall jobs by checking for the existence of that label or get the specific uninstall job by also checking for its value